foundation-faq.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: usage
+title: Quarkus at Commonhaus FAQ
+subtitle: Frequently Asked Questions
+permalink: /foundation/faq
+---
+
+## Why move to a foundation?
+
+Enable us to welcome more contributions from a diverse range of developers and organisations.
```

Comment:
The connection here is a bit weak, to me. Are we trying to say that at the moment we're *prevented* from welcoming contributions (I don't think that's true, but it's kind of implied by the phrasing), or are we saying that we hope having a home outside Red Hat will make contributing to Quarkus more attractive to more people. Or are we saying that if the governance is formally independent of Red Hat more people can become involved in the governance/really get involved in core Quarkus? Or that we want to correct a perception that people outside Red Hat can't contribute? I think we probably want to be saying more of the latter three, and less of the first.